modules/csg/doc_classes/CSGPolygon3D.xml

```diff
@@ -4,71 +4,73 @@
+			When [member mode] is [constant MODE_PATH], if [code]true[/code] the [Transform3D] of the [CSGPolygon3D] is used as the starting point for the extrusions, not the [Transform3D] of the [member path_node].
 		</member>
 		<member name="path_node" type="NodePath" setter="set_path_node" getter="get_path_node">
-			The [Shape3D] object containing the path along which we extrude when [member mode] is [constant MODE_PATH].
+			When [member mode] is [constant MODE_PATH], the location of the [Path3D] object used to extrude the [member polygon].
 		</member>
 		<member name="path_rotation" type="int" setter="set_path_rotation" getter="get_path_rotation" enum="CSGPolygon3D.PathRotation">
-			The method by which each slice is rotated along the path when [member mode] is [constant MODE_PATH].
+			When [member mode] is [constant MODE_PATH], the [enum PathRotation] method used to rotate the [member polygon] as it is extruded.
 		</member>
 		<member name="polygon" type="PackedVector2Array" setter="set_polygon" getter="get_polygon" default="PackedVector2Array(0, 0, 0, 1, 1, 1, 1, 0)">
-			Point array that defines the shape that we'll extrude.
+			The point array that defines the 2D polygon that is extruded.
 		</member>
 		<member name="smooth_faces" type="bool" setter="set_smooth_faces" getter="get_smooth_faces" default="false">
-			Generates smooth normals so smooth shading is applied to our mesh.
+			If [code]true[/code], applies smooth shading to the extrusions.
 		</member>
 		<member name="spin_degrees" type="float" setter="set_spin_degrees" getter="get_spin_degrees">
-			Degrees to rotate our extrusion for each slice when [member mode] is [constant MODE_SPIN].
+			When [member mode] is [constant MODE_SPIN], the total number of degrees the [member polygon] is rotated when extruding.
 		</member>
 		<member name="spin_sides" type="int" setter="set_spin_sides" getter="get_spin_sides">
-			Number of extrusion when [member mode] is [constant MODE_SPIN].
+			When [member mode] is [constant MODE_SPIN], the number of extrusions made.
 		</member>
 	</members>
 	<constants>
 		<constant name="MODE_DEPTH" value="0" enum="Mode">
-			Shape3D is extruded to [member depth].
+			The [member polygon] shape is extruded along the negative z-axis.
 		</constant>
 		<constant name="MODE_SPIN" value="1" enum="Mode">
-			Shape3D is extruded by rotating it around an axis.
+			The [member polygon] shape is extruded by rotating it around the y-axis.
 		</constant>
 		<constant name="MODE_PATH" value="2" enum="Mode">
-			Shape3D is extruded along a path set by a [Shape3D] set in [member path_node].
+			The [member polygon] shape is extruded along the [Path3D] specified in [member path_node].
 		</constant>
 		<constant name="PATH_ROTATION_POLYGON" value="0" enum="PathRotation">
-			Slice is not rotated.
+			The [member polygon] shape is not rotated.
+			[b]Note:[/b] Requires the path z-coordiantes to continually decrease to ensure viable shapes.
```

Comment:
```suggestion
			[b]Note:[/b] Requires the path Z coordinates to continually decrease to ensure viable shapes.
```
(Most important to fix is the typo, but we tend to use "Y axis", "XZ plane" so I think "Z coordinates" might be more consistent with existing docs - I didn't check though.)